### PR TITLE
Update defender-vulnerability-management-capabilities.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the note

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23603

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-vulnerability-management/defender-vulnerability-management-capabilities.md [hunk 1]
```diff
@@ -37,8 +37,7 @@ This article helps clarify the Defender Vulnerability Management capabilities in
 - For new customers or existing Defender for Endpoint P1 or Microsoft 365 E3 customer the **Microsoft Defender Vulnerability Management Standalone is now generally available**. To try it, go to [Try Defender Vulnerability Management Standalone](/microsoft-365/security/defender-vulnerability-management/get-defender-vulnerability-management#try-defender-vulnerability-management-standalone).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-vulnerability-management/defender-vulnerability-management-capabilities.md]:
```suggestion
- For new customers or existing Defender for Endpoint P1 or Microsoft 365 E3 customers the **Microsoft Defender Vulnerability Management Standalone is now generally available**. To try it, go to [Try Defender Vulnerability Management Standalone](/microsoft-365/security/defender-vulnerability-management/get-defender-vulnerability-management#try-defender-vulnerability-management-standalone).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, one additional edit, please.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Gentle reminder.

**Pull request closed without merging**